src/AdvancedHMC.jl

```diff
@@ -7,17 +7,12 @@ using LinearAlgebra: Symmetric, UpperTriangular, mul!, ldiv!, dot, I, diag, chol
 using StatsFuns: logaddexp
 using LazyArrays: BroadcastArray
 using Random: GLOBAL_RNG, AbstractRNG
-using ProgressMeter
+using ProgressMeter: Progress, next!
 using Parameters: @unpack, reconstruct
 using ArgCheck: @argcheck
 
 import StatsBase: sample
 
-# Notations
```

Comment:
I would keep these notations although it is not complete. We can keep improving it.

Reply:
OK I can add them back **with the updated names**.